auslib/web/views/client.py

```diff
@@ -70,6 +72,17 @@ def get(self, **url):
                     response_blobs.append({'product_query': product_query,
                                            'response_release': response_release,
                                            'response_update_type': response_update_type})
+            elif response_blob_names:
+                for blob_name in response_blob_names:
+                    product_query = query.copy()
+                    product_query["product"] = blob_name
```

Comment:
@mozbhearsum 
https://github.com/mozilla/balrog/blob/master/auslib/web/views/client.py#L99
will fail if we do not update the query.